### PR TITLE
LPS-160111 Test Automation Creation - master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-160111

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithContentsPanel.testcase [hunk 1]
```diff
@@ -1483,6 +1483,99 @@ definition {
...
+					value1 = "Page Content");
+
+				AssertTextEquals.assertPartialText(
+					locator1 = "Message#INFO_2",
+					value1 = "There is no content on this page.");
+			}
+
+			task ("View order of buttons on sidebar") {
+				PageEditor.viewSideBarButtonOrder(buttonNameList = "Fragments and Widgets,Browser,Page Design Options,Page Content,Mapping,Comments");
+			}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithContentsPanel.testcase:1575]: View the order of Page Content is the fourth
AssertElementPresent(
);

File: portal-web/test/functional/com/liferay/portalweb/paths/portlet/pageeditor/PageEditor.path [hunk 2]
```diff
@@ -831,6 +831,11 @@
 	<td>//div[contains(@class,'page-contents__content-filter')]//button[contains(@class,'dropdown-toggle')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>PAGE_CONTENTS_DESCRIPTION</td>
+	<td>//p[contains(@class,'page-contents__content-filter')]</td>
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/paths/portlet/pageeditor/PageEditor.path:836]: No need

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithContentsPanel.testcase [hunk 3]
```diff
@@ -1483,6 +1483,99 @@ definition {
...
+					key_text = "paperclip",
+					locator1 = "Icon#ANY");
+			}
+
+			task ("View title and description on Page Content tab") {
+				AssertTextEquals(
+					key_sidebarButton = "Page Content",
+					locator1 = "PageEditor#FRAGMENT_SIDEBAR_TITLE",
+					value1 = "Page Content");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithContentsPanel.testcase:1519]: AssertTextPresent(value1 = "Access the content included in the page and the options to edit them.");

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind updating according to comments? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithContentsPanel.testcase (hunk 1 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithContentsPanel.testcase:1575]: Hi @Tim-Cao! in this case would you like me to remove the macro that I had created and just make the assert on the fourth position? Although it wasn't on the test's scope, couldn't we validate the order of all buttons?

File: portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro [hunk 4]
```diff
@@ -5153,6 +5153,23 @@ return	Array.from(getWebElement("(//div[@aria-label='${fragmentName}'])[${index}
 			value1 = "${numberOfItems}");
 	}
 
+	macro viewSideBarButtonOrder {
+		var index = "1";
+
+		for (var buttonName : list "${buttonNameList}") {
+			var actualButtonValue = selenium.getAttribute("//div[contains(@class,'page-editor__sidebar__buttons')]//button[contains(@data-tooltip-align,'left')][${index}]@title");
+
```
[the remark this answers is not in the document]
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/macros/layout/pageeditor/PageEditor.macro:5161]: Hi Tim! This path doesn't work for our goal on this test, the index will just work when set to 1 which don't verify the right order of items. Could you double-check? For now I will keep as it is and wait your reply.

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind updating according to comments? Thanks

Hi @Tim-Cao, I've updated the PR with some changes you had requested and I will wait your reply about my comment.

**Comment by Tim-Cao** [comment]
See https://github.com/liferay-echo/liferay-portal/pull/9271

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**